types/utils.go

```diff
@@ -75,12 +74,9 @@ func getRootlessRuntimeDirIsolated(env rootlessRuntimeDirEnvironment) (string, e
 		return runtimeDir, nil
 	}
 
-	initCommand, err := ioutil.ReadFile(env.getProcCommandFile())
```

Comment:
- This should have had an extra test case for the new behavior
- After that , `getProcCommandFile` is now unused apart from tests trying to affect it; so drop it, and then remove test cases from the duplicates.

Reply:
(This is assuming that the change is the right thing to do; whether that’s the case is needs to be decided first, and that seems to be an unresolved question so far.)